prompts/configs.go

```diff
@@ -243,7 +255,25 @@ func getValuesForField(field config.SDKGenConfigField, langConfig config.Languag
 		validationMessage = *field.ValidationMessage
 	}
 
-	return true, defaultValue, validationRegex, validationMessage
+	description := ""
+	if field.Description != nil {
+		description = *field.Description
+	}
+	if field.Name == "packageName" && isQuickstart && sdkClassName != "" {
```

Comment:
This way of doing things may break the open close principle. Should I fix that now or is it simple enough for now? @ryan-timothy-albert

Reply:
Ya I think this is fine for now.

Reply:
Not ideal but not worth a massive refactor